src/common/compile.ts

```diff
@@ -242,6 +243,10 @@ export function compileVariant(
   options: CompilationOptions,
   language: Language,
 ): CompilationResult {
+  if (options.level !== "nogolf")
+    try {
+      getOutput(program); // precompute output
+    } catch {}
```

Comment:
What's the point here? I'd think the `hardcode` plugin would end up running `getOutput` anyways, which caches the result.

Reply:
> What's the point here? I'd think the `hardcode` plugin would end up running `getOutput` anyways, which caches the result.

In the past we asserted that there's always only a one compilation going on at a time, but this would run a compilation inside a compilation.